templates/threejs/packages/client/src/mud/createSystemCalls.ts

```diff
@@ -18,6 +44,10 @@ export function createSystemCalls(
     const playerPosition = getComponentValue(Position, playerEntity);
 
     if (playerPosition) {
+      /*
+       * Because MoveSystem is in the root namespace, .moveTo can be called directly
+       * on the World contract.
+       */
```

Comment:
```suggestion
```
`moveTo` isn't part of the MoveSystem or world contract, it's a helper defined above.